modules/log4j2/src/main/java/org/apache/ignite/logger/log4j2/Log4J2Logger.java

```diff
@@ -131,6 +131,11 @@ public Log4J2Logger() {
 
         quiet = quiet0;
         cfg = null;
+
+        // Remove existing handlers attached to j.u.l root logger.
+        Log4jBridgeHandler.removeHandlersForRootLogger();
```

Comment:
As far as I can see, you removed all handlers attached to the j.u.l root registrar. Can you explain why? As I can see from the examples, you can simply add your handler via Log4jBridgeHandler.install().

If I understood this correctly after calling this method, all other handlers (logs appenders) configured by the client, for example, will be removed. Please correct me because I am not close to these methods, but it looks dangerous.

Reply:
This was an optional step from the examples. Let's not install the same handler twice. This can happen if you start/stop a node in the same jvm